bug: failed building go-piper

<!-- Thanks for helping us to improve LocalAI! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->

**LocalAI version:**
Current CI

**Environment, CPU architecture, OS, and Version:**
<!-- Provide the output from "uname -a", HW specs, if it's a VM  -->

**Describe the bug**
Seems to be started recently within the CI - while building the container image. Compilation fails with:


```
 #26 655.6 /usr/local/go/pkg/tool/linux_amd64/link -o $WORK/b001/exe/a.out -importcfg $WORK/b001/importcfg.link -X=runtime.godebugDefault=panicnil=1 -buildmode=exe -buildid=h7hMPg1Dyu87U6l-I3iu/P6-xN_sVDPWnWcq4KpKJ/KXkZZ1XYMeBR6OxAs8WH/h7hMPg1Dyu87U6l-I3iu -extld=g++ $WORK/b001/_pkg_.a
#26 655.9 # github.com/mudler/go-piper/example
#26 655.9 /usr/local/go/pkg/tool/linux_amd64/link: running g++ failed: exit status 1
#26 655.9 /usr/bin/ld: warning: libucd.so, needed by ./piper/build/pi/lib/libespeak-ng.so, not found (try using -rpath or -rpath-link)
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_ispunct'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_properties'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isdigit'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isupper'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isspace'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isalpha'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_toupper'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isalnum'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_lookup_category'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_tolower'
#26 655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_islower'
#26 655.9 collect2: error: ld returned 1 exit status
#26 655.9 
#26 655.9 make[1]: *** [Makefile:32: example/main] Error 1
#26 655.9 make[1]: Leaving directory '/build/sources/go-piper'
#26 655.9 make: *** [Makefile:223: sources/go-piper/libpiper_binding.a] Error 2
#26 ERROR: process "/bin/sh -c make build" did not complete successfully: exit code: 2
------
 > [builder 7/8] RUN make build:
655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_toupper'
655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_isalnum'
655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_lookup_category'
655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_tolower'
655.9 /usr/bin/ld: ./piper/build/pi/lib/libespeak-ng.so: undefined reference to `ucd_islower'
655.9 collect2: error: ld returned 1 exit status
655.9 
655.9 make[1]: *** [Makefile:32: example/main] Error 1
655.9 make[1]: Leaving directory '/build/sources/go-piper'
655.9 make: *** [Makefile:223: sources/go-piper/libpiper_binding.a] Error 2
```

**To Reproduce**
<!-- Steps to reproduce the behavior, including the LocalAI command used, if any -->

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Logs**
https://github.com/mudler/LocalAI/actions/runs/7015798457/job/19099206160

**Additional context**
N/A

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ WHISPER_CPP_VERSION?=85ed71aaec8e0612a84c0b67804bde75aa75a273
 BERT_VERSION?=6abe312cded14042f6b7c3cd8edf082713334a4d
 
 # go-piper version
-PIPER_VERSION?=7fe05263b4ca3ffa93a53e2737643a6a6afb9a7b
+PIPER_VERSION?=5a4c9e28c84bac09ab6baa9f88457d852cb46bb2
 
 # stablediffusion version
 STABLEDIFFUSION_VERSION?=902db5f066fd137697e3b69d0fa10d4782bd2c2f
@@ -126,7 +126,7 @@ ifeq ($(findstring tts,$(GO_TAGS)),tts)
 #	OPTIONAL_TARGETS+=go-piper/libpiper_binding.a
 #	OPTIONAL_TARGETS+=backend-assets/espeak-ng-data
 	PIPER_CGO_CXXFLAGS+=-I$(shell pwd)/sources/go-piper/piper/src/cpp -I$(shell pwd)/sources/go-piper/piper/build/fi/include -I$(shell pwd)/sources/go-piper/piper/build/pi/include -I$(shell pwd)/sources/go-piper/piper/build/si/include
- 	PIPER_CGO_LDFLAGS+=-L$(shell pwd)/sources/go-piper/piper/build/fi/lib -L$(shell pwd)/sources/go-piper/piper/build/pi/lib -L$(shell pwd)/sources/go-piper/piper/build/si/lib -lfmt -lspdlog
+ 	PIPER_CGO_LDFLAGS+=-L$(shell pwd)/sources/go-piper/piper/build/fi/lib -L$(shell pwd)/sources/go-piper/piper/build/pi/lib -L$(shell pwd)/sources/go-piper/piper/build/si/lib -lfmt -lspdlog -lucd
 	OPTIONAL_GRPC+=backend-assets/grpc/piper
 endif
 

